policies/k8srequiredlabels.rego

```diff
@@ -0,0 +1,11 @@
+package k8srequiredlabels
```

Comment:
remove this file, it serves no use case

Reply:
we have decide which policies to activate. Every file I've added here was tested against the `contraints.yaml` values to understand `.rego` better.